addons/modules/functions/fnc_moduleHideTerrainObjects.sqf

```diff
@@ -0,0 +1,52 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to hide terrain objects.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call zen_modules_fnc_moduleHideTerrainObjects
+ *
+ * Public: No
+ */
+
+params ["_logic"];
+
+[localize "STR_a3_to_hideTerrainObjects1", [
+    ["TOOLBOX:YESNO", "STR_a3_to_hideTerrainObjects1", true, true],
```

Comment:
Why force default on this one only? I think it would be fine to save selection for this too.

Reply:
3d3bb13